content/_index.html

```diff
@@ -15,8 +15,11 @@
 	<a class="btn btn-lg btn-secondary mr-3 mb-4" target="_blank" href="/FSHOnline/">
 		Play with FSH <i class="fas fa-fish "></i>
 	</a>
+	<a class="btn btn-lg btn-info mr-3 mb-4" target="_blank" href="/courses/">
+		On-Line Course <i class="fas fa-graduation-cap "></i>
+	</a>
 	<a class="btn btn-lg btn-danger mr-3 mb-4" target="_blank" href="/fsh-finder/">
-		Who's on Board? <i class="fas fa-ship "></i>
+		Who else is FSHing? <i class="fas fa-ship "></i>
```

Comment:
```suggestion
		Who Else Is FSHing? <i class="fas fa-ship "></i>
```

I think all these buttons are Title Case? https://individed.com/code/to-title-case/ says `Else` and `Is` should be capitalized. (I don't feel strongly they should be Title Case, but I do feel strongly about consistency 😃.)